src/main/java/stsjorbsmod/patches/AbstractCardMakeStatEquivalentCopyPatch.java

```diff
@@ -31,8 +31,13 @@ public static AbstractCard Postfix(AbstractCard __result, AbstractCard __this) {
             card.baseMetaMagicNumber = source.baseMetaMagicNumber;
             card.isMetaMagicNumberModified = source.isMetaMagicNumberModified;
             card.upgradedMetaMagicNumber = source.upgradedMetaMagicNumber;
+            boolean entombed = EntombedField.entombed.get(source);
```

Comment:
Would be a little less redundant to do this regardless of the `if` block (similar to the `WrathField` at the top of this method)